config/locales/static/whats_new.yml

```diff
@@ -0,0 +1,72 @@
+---
+en:
+  static:
+    whats_new:
+      heading: What's new in the training
+      content: |
+        <div class="govuk-inset-text">
+          We’ve added a few things since you were last here. 
+        </div>
+
+        <h2>Videos</h2>
```

Comment:
why not `##`?

Reply:
It didn't seem to work after the first $ syntax so i changed it all to HTML for consistency. Perhaps it was something i was doing wrong. I'll try again